### PR TITLE
Document scope option for the init method

### LINKED ISSUE #1509: Document scope option for the init method
### Description

Keycloak.js supports specifying explicit scopes in keycloak.init(), but the option is missing from the documentation.

Adapter being documented:
https://github.com/keycloak/keycloak/blob/90fe97133c5b5474286b56121679393e46ff01df/adapters/oidc/js/src/keycloak.js#L152

### PR DESCRIPTION
**Pull request by epuronta**
Keycloak.js supports specifying explicit scopes in `keycloak.init()`, but the option is missing from the documentation. This PR adds that documentation.

Adapter being documented:
https://github.com/keycloak/keycloak/blob/90fe97133c5b5474286b56121679393e46ff01df/adapters/oidc/js/src/keycloak.js#L152

Closes #1509

### CODE REVIEW

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 1]
```diff
@@ -435,6 +435,10 @@ Options is an Object, where:
 * token - Set an initial value for the token.
 * refreshToken - Set an initial value for the refresh token.
 * idToken - Set an initial value for the id token (only together with token or refreshToken).
+* scope - Set the default scope parameter to the {project_name} login endpoint. Use a space-delimited list of scopes. Those typically
+reference link:{adminguide_link}#_client_scopes[Client scopes] defined on particular client. Note that the scope `openid` will be
```
Comment (andymunro) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
reference link:{adminguide_link}#_client_scopes[Client scopes] defined on a particular client. Note that the scope `openid` will be
```

**Review by andymunro — CHANGES_REQUESTED** [review]
@epuronta One minor change to suggest. Thanks.

File: securing_apps/topics/oidc/javascript-adapter.adoc (hunk 1 above)
Comment (epuronta) [reply securing_apps/topics/oidc/javascript-adapter.adoc]: Thanks, fixed the missing article

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 2]
```diff
@@ -435,6 +435,10 @@ Options is an Object, where:
 * token - Set an initial value for the token.
 * refreshToken - Set an initial value for the refresh token.
 * idToken - Set an initial value for the id token (only together with token or refreshToken).
+* scope - Set the default scope parameter to the {project_name} login endpoint. Use a space-delimited list of scopes. Those typically
+reference link:{adminguide_link}#_client_scopes[Client scopes] defined on a particular client. Note that the scope `openid` will be
```
Comment (stianst) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
reference link:{adminguide_link}#_client_scopes[Client scopes] defined on a particular client. Note that the scope `openid` will
```

**Pull request merged**